### PR TITLE
feat: display modeline as a thin bar in GUI

### PR DESCRIPTION
**Pull request by AmaiKinono**
As mentioned in https://github.com/kiennq/emacs-mini-modeline/issues/18.

### CODE REVIEW

File: mini-modeline.el [hunk 1]
```diff
@@ -80,8 +80,28 @@ Will be set if `mini-modeline-enhance-visual' is t."
   :type 'boolean
   :group 'mini-modeline)
 
+(defface mini-modeline-active-window-modeline
```
Comment (kiennq) [inline mini-modeline.el]: nit: To consistent with original face `mode-line`, I think we should change the name of this to `mini-modeline-mode-line` and the one bellow to `mini-modeline-mode-line-inactive`.

File: mini-modeline.el [hunk 2]
```diff
@@ -80,8 +80,28 @@ Will be set if `mini-modeline-enhance-visual' is t."
...
+     :background "#dddddd" :height 0.1 :box nil)
+    (t
+     :background "#333333" :height 0.1 :box nil))
+  "Modeline face for inactive window."
+  :group 'mini-modeline)
+
 (defvar mini-modeline--orig-mode-line mode-line-format)
 (defvar mini-modeline--echo-keystrokes echo-keystrokes)
+(defvar mini-modeline--orig-mode-line-remap
+  (alist-get 'mode-line face-remapping-alist))
```
Comment (kiennq) [inline mini-modeline.el]: This should be `(or (alist-get 'mode-line face-remapping-alist) 'mode-line)`, so we will correctly restore the original face when disable `mini-modeline`. Same for bellow.

File: mini-modeline.el (hunk 1 above)
Comment (AmaiKinono) [reply mini-modeline.el]: Done.

File: mini-modeline.el (hunk 2 above)
Comment (AmaiKinono) [reply mini-modeline.el]: Done.

**Pull request merged**